dogfood.sh

```diff
@@ -0,0 +1,28 @@
+if [ -z "$ZSH_VERSION" ]; then
+  source="${BASH_SOURCE[0]}"
+  # resolve $SOURCE until the file is no longer a symlink
+  while [[ -h $source ]]; do
+    scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+    source="$(readlink "$source")"
+
+    # if $source was a relative symlink, we need to resolve it relative to the path where the
+    # symlink file was located
+    [[ $source != /* ]] && source="$scriptroot/$source"
+  done
+  scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+else
+  # :A will resolve all symlinks, :h will truncate last path component leaving you with a directory name
+  scriptroot=${0:A:h}
+fi
+
+REPO_ROOT=$(cd "${scriptroot}";pwd)
+
+PKG_DIR=${REPO_ROOT}/artifacts/packages/Release/Shipping
+if [ ! -d "$PKG_DIR" ]; then
+  PKG_DIR=${REPO_ROOT}/artifacts/packages/Debug/Shipping
+fi
+
+export PATH=$REPO_ROOT/artifacts/bin/dotnet-latest:$PATH
+echo Setting export BUILT_NUGETS_PATH=$PKG_DIR to resolve locally built packages
+export BUILT_NUGETS_PATH=$PKG_DIR
+echo "Use $REPO_ROOT/tests/Aspire.Workload.Tests/data/nuget8.config as your local nuget.config to ensure the packages can be restored"
```

Comment:
Do we need a dogfood.ps1 for Windows too?

Reply:
Yes. Coming soon in the next PR 😬

Reply:
https://github.com/dotnet/aspire/issues/4222